installation/install.sh

```
@@ -2488,6 +2492,8 @@ install_db(){
     /bin/rm -rf $ssh_tmp_dir
     mkdir -p $ssh_tmp_dir
     show_spinner cs_gen_sshkey $ssh_tmp_dir
+    #ensure sshd is running before ansible connects via ssh (alinux4 may not have sshd started at firstboot)
+    systemctl is-active sshd >/dev/null 2>&1 || systemctl start sshd >>$ZSTACK_INSTALL_LOG 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)